### ISSUE TITLE
 Feature Request: Detect `src="{{ var }}"` Instead of `[src]="var"`

### ISSUE THREAD

**Issue by jon9090** [angular-eslint/angular-eslint#1921]
**Description:**
I would like to request a new feature for Angular ESLint that detects instances of `src="{{ var }}"` in templates and recommends replacing them with `[src]="var"`. This will help ensure best practices are followed by using Angular's property binding instead of string interpolation.

**Motivation:**
Using `src="{{ var }}"` can lead to potential issues, especially when dealing with URLs that require proper binding for security and performance reasons. Property binding `[src]="var"` ensures that Angular can efficiently handle the binding and updates.

**Suggested Implementation:**
1. Add a new rule to the ESLint configuration that identifies occurrences of `src="{{ var }}"`.
2. Provide an appropriate error or warning message indicating that property binding should be used instead.
3. Suggest the correct usage `[src]="var"`.

**Example:**
For instance, the rule should flag the following code:

```html
<img src="{{ imageUrl }}">
```

And suggest the following correction:

```html
<img [src]="imageUrl">
```

**Additional Context:**
Ensuring that property binding is used instead of string interpolation for attributes like `src` will help maintain consistency and follow Angular best practices.

Thank you for considering this feature request!

**Comment by wizardnet972** [comment]
Check: @angular-eslint/template/no-interpolation-in-attributes
